app/forms/candidate_interface/pick_course_form.rb

```diff
@@ -33,9 +33,8 @@ def dropdown_available_courses
                    course.name_code_and_provider
                  elsif courses_with_name_description_provider_and_age_range.count(course.name_description_provider_and_age_range.downcase) == 1 && course.age_range.present?
                    course.name_code_and_age_range
-                 else
-                   course.name_and_code
                  end
+          name ||= course.name_and_code
```

Comment:
This one seems a bit odd, is it enforced by the cop?

Reply:
The cop was complaining about the duplicate branch, but I couldn't remove the check on line 28 as the checks need to run in a specific order so this was the best alternative I could think of.

Reply:
Oh ok, maybe we could extract this decision into a separate method then?

Reply:
Ok, I'll look into it when I get a minute!